mantis-control-plane/mantis-control-plane-server/src/main/java/io/mantisrx/master/resourcecluster/proto/ResourceClusterAPIProto.java

```diff
@@ -17,25 +17,32 @@
 package io.mantisrx.master.resourcecluster.proto;
 
 import io.mantisrx.master.jobcluster.proto.BaseResponse;
+import io.mantisrx.shaded.com.fasterxml.jackson.annotation.JsonCreator;
+import io.mantisrx.shaded.com.fasterxml.jackson.annotation.JsonProperty;
 import java.util.List;
 import lombok.Builder;
 import lombok.Singular;
 import lombok.Value;
 
 public class ResourceClusterAPIProto {
 
+    // TODO: @Value generated equals doesn't include base class fields.
     @Value
     public static class ListResourceClustersResponse extends BaseResponse {
```

Comment:
are there tests to verify the structure of these Jackson serialized data structures? If not, can we add them?

Reply:
Added another TODO in one of the tests that covers this.